strawberry/types/fields/resolver.py

```diff
@@ -18,6 +18,10 @@
 T = TypeVar("T")
 
 
+def default_resolver(name: str, source: Any, *args, **kwargs):
```

Comment:
I think this should live as an attribute on `StrawberryResolver`, probably private